app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    # Check each friend using try/except
+    for friend in friends:
+        try:
+            # Try to allow the friend to visit the cafe
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            # Catch both NotVaccinatedError and OutdatedVaccineError
+            return "All friends should be vaccinated"
```

Comment:
The return statement here should be inside the loop only if you want to exit the function immediately when a friend is not vaccinated. However, if you want to check all friends and then decide, consider using a flag or a different approach to accumulate the result.